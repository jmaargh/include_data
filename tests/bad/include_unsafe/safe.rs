fn main() {}

static FOO: u64 = include_data::include_unsafe!("../../test_data/binary_8");
