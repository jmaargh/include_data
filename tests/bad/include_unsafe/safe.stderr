error[E0133]: call to unsafe function is unsafe and requires unsafe function or block
 --> tests/bad/include_unsafe/safe.rs:3:19
  |
3 | static FOO: u64 = include_data::include_unsafe!("../../test_data/binary_8");
  |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ call to unsafe function
  |
  = note: consult the function's documentation for information on how to avoid undefined behavior
  = note: this error originates in the macro `include_data::include_unsafe` (in Nightly builds, run with -Z macro-backtrace for more info)
